hr_expense_invoice/views/hr_expense_views.xml

```diff
@@ -1,111 +1,98 @@
+<?xml version="1.0" encoding="utf-8" ?>
 <odoo>
     <record id="hr_expense_view_form" model="ir.ui.view">
         <field name="name">hr.expense.form</field>
         <field name="model">hr.expense</field>
         <field name="inherit_id" ref="hr_expense.hr_expense_view_form" />
         <field name="arch" type="xml">
-            <field name="unit_amount" position="attributes">
-                <attribute name="force_save">1</attribute>
+            <field name="price_unit" position="attributes">
+                <attribute name="force_save">
+                    1
```

Comment:
Check if this could be in te same line

If not, prefer `True`

Same in similar cases

Reply:
done, same line